.github/workflows/build.yml

```diff
@@ -33,8 +33,9 @@ jobs:
 
     - name: Install R support
       run: |
-        sudo apt-get install r-base-dev
-        sudo R -e 'install.packages(c("reticulate", "IRkernel"))'
+        sudo apt-get remove r-base
```

Comment:
Changing this to `sudo apt-get remove r-base-core` worked for me, though it still took c. 4min to build the whole thing. Weirdly the r-base-core version already installed is v4.0.5 which is incompatible with the r-cran-reticulate version that gets installed here. But when you reinstall it (via the r-cran-reticulate dependency), it installs v3.6.3 which is apparently OK. See [build](https://github.com/brieuclehmann/tutorials/runs/2479369702).